frontend/src/scenes/scenes.ts

```diff
@@ -489,6 +489,8 @@ export const routes: Record<string, Scene> = {
     [urls.insightView(':shortId' as InsightShortId)]: Scene.Insight,
     [urls.insightSubcriptions(':shortId' as InsightShortId)]: Scene.Insight,
     [urls.insightSubcription(':shortId' as InsightShortId, ':subscriptionId')]: Scene.Insight,
+    [urls.alert(':shortId' as InsightShortId, ':subscriptionId')]: Scene.Insight,
```

Comment:
subscriptionId should be renamed to itemId here and in other similar places, but I don't want to add more changes to this PR, probably will do in a following PR